src/__template__/ComponentTemplate.vue

```diff
@@ -98,6 +98,7 @@ const rootElementStyles = computed((): Record<string, string> => ({
   height: iconSize.value,
   lineHeight: '0',
   width: iconSize.value,
+  pointerEvents: props.decorative ? 'none' : 'all',
```

Comment:
Instead of explicitly setting to `all` here (which would override any rules in the host app) just fallback to `undefined` so that no rule is applied.

Should it have a tabindex of -1 as well?

Reply:
Good call. Updated.

Reply:
Just make sure that `undefined` doesn't eval to `pointer-events: undefined` in the browser (if so, use `null` instead)

Reply:
Updated & tested -- works as expected.